ceph.spec.in

```diff
@@ -158,7 +158,11 @@ BuildRequires:	gcc-c++
 %endif
 BuildRequires:	gdbm
 %if 0%{with tcmalloc}
-%if 0%{?fedora} || 0%{?rhel}
+# libprofiler did not build on ppc64le until 2.7.90
+%if 0%{?fedora} || 0%{?rhel} >= 8
+BuildRequires:	gperftools-devel >= 2.7.90
```

Comment:
now that gperftools 2.7 in EPEL builds on these platforms (https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2021-0eda4297eb), is this version requirement correct?

Reply:
@ktdreyer ^

Reply:
Note that this same code is also in master, so if it is going to be changed we would need to do it there, first.